pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/CollectiveDataEdition/__specs__/CollectiveDataEdition.spec.tsx

```diff
@@ -302,6 +337,11 @@ describe('CollectiveDataEdition', () => {
         </Provider>
       )
 
+      // wait for loader to disappear
+      await waitFor(() => {
+        expect(screen.getByLabelText(/E-mail/)).toBeInTheDocument()
+      })
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion(non-bloquant):** ‏Mettre ça dans une petite fonction `waitForLoader` pourrait être moins verbeux